def f(n: int):
    x = 4

f(5)
print(x)  # Error: x is in function f's scope